### PR TITLE
Fixes announcement bar styles

### PR DESCRIPTION
**Pull request by mike-lvov**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/330
closes https://github.com/sparkletown/internal-sparkle-issues/issues/329

### CODE REVIEW

File: src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss [hunk 1]
```diff
@@ -6,19 +6,20 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  left: 50%;
+  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
+  left: 40%;
```
Comment (0xdevalias) [inline src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:10]: Is it possible to set a variable for the sidebar/map %'s, and then calculate the value here by using that variable divide 2?

Comment (mike-lvov) [reply src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:10]: I think yes, should take not much time, but the sidebar's width/logic will change in https://github.com/sparkletown/sparkle/pull/1074

**Review by 0xdevalias — APPROVED** [review]
My previous comment isn't really blocking, so while it would be nice to have, not worth holding approval over.

File: src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss [hunk 2]
```diff
@@ -6,19 +6,20 @@
...
   width: 500px;
```
Comment (denisdimitrov) [inline src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:11]: You say it takes 20%, how about this width value?

Comment (denisdimitrov) [reply src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:11]: Looks like it has 500px width and -250 margin.

Comment (mike-lvov) [reply src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:11]: I believe this is a technique to make the bar centered. This width + margin styles set the `moving point` of the bar to the center of the bar

File: src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss (hunk 1 above)
Comment (mike-lvov) [reply src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss]: fixed in https://github.com/sparkletown/sparkle/pull/1156/commits/d46487a0e6597ec71501e136b1822c8586897a0b

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss [hunk 3]
```diff
@@ -6,19 +6,20 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  left: 50%;
+  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
```
Comment (0xdevalias) [inline src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss:9]: @mike-lvov What does this '40%' here refer to? It doesn't seem to align with the code? Is it outdated/can it be deleted?

**Pull request merged**